src/main.ts

```diff
@@ -106,7 +107,7 @@ export default class LinterPlugin extends Plugin {
         const startMessage = 'This will edit all of your files and may introduce errors.';
         const submitBtnText = 'Lint All';
         const submitBtnNoticeText = 'Linting all files...';
-        new LintConfirmationModal(this.app, startMessage, submitBtnText, submitBtnNoticeText, () =>{
+        new LintConfirmationModal(this.app, startMessage, submitBtnText, submitBtnNoticeText, () => {
```

Comment:
Are you using the eslint rules to make these changes or is this your editor/IDE that is making the spacing changes? I am curious since this will just be undone if it is not being created by eslint.

Reply:
This is the change I made when I ran `npm run lint`, and I don't know why it's different from yours.

Reply:
Gotcha. It could be that it is doing something weird on my OS. But this should be fine then.